test/test-tcp-write-ready.c

```diff
@@ -0,0 +1,210 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
I think this should just be libuv collaborators, but double check a recently added file.

Reply:
Thanks, Fixed